packages/utils/src/object.ts

```diff
@@ -46,12 +47,12 @@ export function objectMap (source, mapping, thisArg) {
   }
   return target
 }
-export function getObjectOwnProperty (obj, propName) {
+export function getObjectOwnProperty (obj, propName: string) {
   return hasOwnProperty(obj, propName) ? obj[propName] : undefined
 }
 
-export function clonePlainObjectDeep (obj, seen) {
-  if (!seen) { seen = [] }
+export function clonePlainObjectDeep (obj, seen?: any[]) {
+  if (!seen) { seen = new Array() }
```

Comment:
aside: we could/should improve the perf. here with a Set instead of array for `seen`

Reply:
Yes, even just `/** @deprecated */` (in a subsequent PR) would be nice here. It's _possible_ someone is using it externally but it's not a core deliverable of TKO/knockout and trivial to do on your own so I think it'd be good to get rid of it if it's not being used internally.

Reply:
Done